### PR TITLE
Set parameter for Raw Data Exporter in Pupil Player #1239

### PR DESCRIPTION
**Pull request by fneitzel**
I didn't know how to claim the task ;)

Integrated Switches.

### CODE REVIEW

File: pupil_src/shared_modules/raw_data_exporter.py [hunk 1]
```diff
@@ -105,12 +105,24 @@ class Raw_Data_Exporter(Analysis_Plugin_Base):
         '''
     icon_chr = chr(0xe873)
     icon_font = 'pupil_icons'
+    
+    def __init__(self, g_pool, export_pupil_positions=True, export_pupil_gaze_positions_info=True, export_gaze_positions=True):
+        super().__init__(g_pool)
+        self.export_pupil_positions = export_pupil_positions
+        self.export_pupil_gaze_positions_info = export_pupil_gaze_positions_info
+        self.export_gaze_positions = export_gaze_positions
```
Comment (papr) [inline pupil_src/shared_modules/raw_data_exporter.py]: Would it be possible to rename these to `should_export_*`? Also, may I suggest to rename `export_pupil_gaze_positions_info` to `should_export_field_info`?

**Comment by papr** [comment]
I caught a small undefined variable name bug. I applied the black code format and removed the `== True` while I was at it.

Thank you again @fneitzel, for contributing and investing your time into this project!

File: pupil_src/shared_modules/raw_data_exporter.py [hunk 2]
```diff
@@ -102,162 +102,254 @@ class Raw_Data_Exporter(Analysis_Plugin_Base):
...
+
     icon_chr = chr(0xe873)
-    icon_font = 'pupil_icons'
+    icon_font = "pupil_icons"
+
+    def __init__(
+        self,
+        g_pool,
+        should_export_pupil_positions=True,
+        should_export_field_info=True,
```
Comment (mkassner) [inline pupil_src/shared_modules/raw_data_exporter.py]: I know this is terribly late but I wonder. The initial motivation for adding this complexity was that exports a large/take a long time. This reason does not apply to writing the filed info. Why is this a setting then. Whats the cost of always doing this?

Comment (mkassner) [reply pupil_src/shared_modules/raw_data_exporter.py]: @papr

Comment (mkassner) [reply pupil_src/shared_modules/raw_data_exporter.py]: I know this is nit-picky but I just want to drive home the point that every slider and setting comes at a cost and need justification.

**Pull request merged**